queries.sql

```diff
@@ -45,3 +45,10 @@ SELECT AVG(weight_kg) FROM animals;
 SELECT CASE WHEN (COUNT(CASE WHEN neutered THEN 1 END)) > COUNT(CASE WHEN not neutered THEN 1 END) THEN 'NEUTERED' ELSE 'NOT NEUTERED' END FROM animals;
 SELECT species, MIN(weight_kg) as "Minimum Weight", MAX(weight_kg) as "Maximum Weight" FROM animals GROUP BY species;
 SELECT species, AVG(escape_attempts) as "AVERAGE ESCAPE ATTEMPTS" from animals WHERE date_of_birth BETWEEN '1990-01-01' and '2000-01-01' GROUP BY species;
+SELECT * FROM animals WHERE owner_id=(SELECT id FROM owners WHERE full_name='Melody');
```

Comment:
- [ ] Please join the animal and owner table so the query works as expected.
  - You can consider using this code. `SELECT * FROM animals LEFT JOIN owners ON animals.owner_id = owners.id WHERE full_name = 'Melody Pond';`